From 89fc82799e9c983873c61e5cef59217b0dd5e4bc Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Thu, 16 Jan 2025 15:36:33 +0100
Subject: [PATCH] media: stm32: csi: removal dead code within stm32_csi_start

Remove the assignment of unused lanes_stop variable.

Change-Id: I9748dcf8c18893c140fd996abbf0e6f602e5d289
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/429388
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 drivers/media/platform/st/stm32/stm32-csi.c | 3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/media/platform/st/stm32/stm32-csi.c
+++ b/drivers/media/platform/st/stm32/stm32-csi.c
@@ -466,7 +466,6 @@ static int stm32_csi_start(struct stm32_
 	int ret, i, mbps;
 	u32 lanes_ie = 0;
 	u32 lanes_en = 0;
-	u32 lanes_stop = 0;
 	u32 ccfr;
 	s64 link_freq;
 
@@ -510,11 +509,9 @@ static int stm32_csi_start(struct stm32_
 		if (!csi2priv->lanes[i]) {
 			lanes_ie |= STM32_CSI_SR1_DL0_ERRORS;
 			lanes_en |= STM32_CSI_PCR_DL0EN;
-			lanes_stop |= STM32_CSI_SR1_STOPDL0F;
 		} else {
 			lanes_ie |= STM32_CSI_SR1_DL1_ERRORS;
 			lanes_en |= STM32_CSI_PCR_DL1EN;
-			lanes_stop |= STM32_CSI_SR1_STOPDL1F;
 		}
 	}
 
